api/apps/graphql/resolvers.coffee

```diff
@@ -1,11 +1,17 @@
 _ = require 'underscore'
 { where, presentCollection } = Article = require '../articles/model'
+User = require '../users/model.coffee'
+
+module.exports.articles = (root, args, req, ast) ->
+  if (not args.published or args.scheduled) and not args.channel_id
```

Comment:
👍  nice. I wonder if it'd make sense to abstract these permissions to work based on GraphQL args and query params (probably a leaky abstraction, but food for thought).